src/main/java/com/sprint/mission/discodeit/controller/AuthController.java

```diff
@@ -49,32 +97,119 @@ public ResponseEntity<?> getUser(Authentication authentication) {
             return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("인증 정보가 올바르지 않습니다.");
         }
         return ResponseEntity.ok(userMapper.toDto(userDetails.getUser()));
+    }*/
+
+    @GetMapping("/me")
+    public ResponseEntity<String> getAccessTokenFromRefreshToken(HttpServletRequest request) {
+        Cookie[] cookies = request.getCookies();
+        if (cookies == null) {
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("No cookies found");
+        }
+
+        String refreshToken = null;
+        for (Cookie cookie : cookies) {
+            if ("refresh_token".equals(cookie.getName())) {
+                refreshToken = cookie.getValue();
+                break;
+            }
+        }
```

Comment:
`@CookieValue` 애노테이션 사용하면 코드가 훨씬 간결해질 것 같아요!  

```suggestion
    @GetMapping("/me")
    public ResponseEntity<String> getAccessTokenFromRefreshToken(@CookieValue(value = "refresh_token") String refreshToken) {
```